pkg/quarterdeck/config/config.go

```diff
@@ -27,6 +27,7 @@ type Config struct {
 	LogLevel      logger.LevelDecoder `split_words:"true" default:"info"`                          // $QUARTERDECK_LOG_LEVEL
 	ConsoleLog    bool                `split_words:"true" default:"false"`                         // $QUARTERDECK_CONSOLE_LOG
 	AllowOrigins  []string            `split_words:"true" default:"http://localhost:3000"`         // $QUARTERDECK_ALLOW_ORIGINS
+	InviteBaseURL string              `split_words:"true" default:"https://rotational.app/invite"` // $QUARTERDECK_INVITE_BASE_URL
```

Comment:
Can we put this in its own configuration and separate them out it to components to make adding links easier in the future? Something like:

```go
type EmailURLsConfig struct {
    Base string `default:"https://rotational.app"`
    Verify string `default:"/verify"`
    Invite  string `deafult:"/invite"` 
}

func (c EmailURLsConfig) VerifyURL() *url.URL {
    u, _ := url.Parse(c.Base)
    return u.ResolveReference(url.URL{Path: c.Verify})
}

// etc
… 
```

Reply:
This will also make it easier to put the params on and you can pass just the `EmailURLsConfig` to the email manager rather than the whole config (you may have to pass two configs, one for the email manager settings and one for the URLs -- but the second one could be an interface rather than a config object to make testing easier).